### PR TITLE
Update windows-autopilot-hybrid.md

### PR DESCRIPTION
**Pull request by msbemba**
Created a new section for prereqs to install intune connector as it was diluted  in the installation steps

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by PRMerger7** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2807

File: memdocs/autopilot/windows-autopilot-hybrid.md [hunk 1]
```diff
@@ -113,7 +113,14 @@ The organizational unit that's granted the rights to create computers must match
 
 ## Install the Intune Connector
 
-The Intune Connector for Active Directory must be installed on a computer that's running Windows Server 2016 or later. The computer must also have access to the internet and your Active Directory. To increase scale and availability, you can install multiple connectors in your environment. We recommend installing the Connector on a server that's not running any other Intune connectors. Each connector must be able to create computer objects in any domain that you want to support.
+## Prerequisites
```
Comment (JohanFreelancer9) [inline memdocs/autopilot/windows-autopilot-hybrid.md:116]:
```suggestion
### Prerequisites
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger15** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dougeby Please review and signoff this PR. Thanks

File: memdocs/autopilot/windows-autopilot-hybrid.md [hunk 2]
```diff
@@ -113,7 +113,14 @@ The organizational unit that's granted the rights to create computers must match
 
 ## Install the Intune Connector
 
-The Intune Connector for Active Directory must be installed on a computer that's running Windows Server 2016 or later. The computer must also have access to the internet and your Active Directory. To increase scale and availability, you can install multiple connectors in your environment. We recommend installing the Connector on a server that's not running any other Intune connectors. Each connector must be able to create computer objects in any domain that you want to support.
+### Prerequisites
+
+- The Intune Connector for Active Directory must be installed on a computer that's running Windows Server 2016 or later. 
+- The computer must also have access to the internet and your Active Directory.
+- To increase scale and availability, you can install multiple connectors in your environment.
+- We recommend installing the Connector on a server that's not running any other Intune connectors.
```
Comment (JohanFreelancer9) [inline memdocs/autopilot/windows-autopilot-hybrid.md:121]:
```suggestion
- We recommend installing the connector on a server that's not running any other Intune connectors.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba, one more, please.

**Comment by PRMerger9** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by frankroj** [comment]
We do not want to duplicate prerequisites in two different sections of the same article. Instead I am going to split the prerequisites into two sections - client prerequisites and Intune connector server prerequisites, and then list all prerequisites at the top. Down in the Install section I will then link back to the Intune connector server prerequisites to remind the user. Due to the many changes involved and this branch having a conflict, I am closing this PR and will create a new PR for these changes.

**Comment by frankroj** [comment]
Replaced with PR https://github.com/MicrosoftDocs/memdocs-pr/pull/9501

**Pull request closed without merging**